crates/ruff/src/rules/flake8_bugbear/rules/assert_raises_exception.rs

```diff
@@ -7,14 +7,14 @@ use ruff_python_ast::types::Range;
 use crate::checkers::ast::Checker;
 
 /// ## What it does
-/// Checks for `self.assertRaises(Exception)`.
+/// Checks for `self.assertRaises(Exception)` or `pytest.raises(Exception)`.
 ///
 /// ## Why is this bad?
-/// `assertRaises(Exception)` can lead to your test passing even if the
-/// code being tested is never executed due to a typo.
+/// These forms catch every Exception, which can lead to passing tests even
+/// the code being tested is never executed due to a typo.
 ///
-/// Either assert for a more specific exception (builtin or custom), use
-/// `assertRaisesRegex` or the context manager form of `assertRaises`.
```

Comment:
This felt misleading, since the context manager form of `assertRaises` actually gives a lint error!